frontend/packages/console-shared/src/selectors/pod.ts

```diff
@@ -3,3 +3,8 @@ import { PodKind } from '@console/internal/module/k8s';
 
 export const getNodeName = (pod: PodKind) =>
   get(pod, 'spec.nodeName') as PodKind['spec']['nodeName'];
+
+export const getPodContainers = (pod: PodKind) =>
```

Comment:
Why not just use `pod.spec.containers` instead of `getPodContainer(pod)`, etc.? These helpers aren't that helpful imo. `pod.spec.containers` is a required field, so it's always set.

Reply:
This is a pattern I'd rather not encourage, although I know it's crept into our code in a number of places.

Reply:
You also lose type checking with lodash get